# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/vr/ipc/PVR.ipdl
# Commit: f92f2cc29cb1
# Full Hash: f92f2cc29cb1e91274550f642affab21783ecfd5
# Author: Jean-Yves Avenard <jyavenard@mozilla.com>
# Date: 2019-05-25 09:39:13
# Regressor Bug: 1550422
# File Overlap Count: 2
# Description:
#   Bug 1550422 - P9. Sync preferences in VR process when they change. r=daoshengmu
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D31017
# ==============================================================================

diff -r 6dc82f88333d -r f92f2cc29cb1 gfx/vr/ipc/PVR.ipdl
--- a/gfx/vr/ipc/PVR.ipdl	Thu May 23 04:13:07 2019 +0000
+++ b/gfx/vr/ipc/PVR.ipdl	Thu May 23 04:13:07 2019 +0000
@@ -10,6 +10,7 @@
 
 include GraphicsMessages;
 include MemoryReportTypes;
+include PrefsTypes;
 include protocol PVRGPU;
 
 include "VRMessageUtils.h";
@@ -26,6 +27,7 @@
 
   async UpdatePref(GfxPrefSetting pref);
   async UpdateVar(GfxVarUpdate var);
+  async PreferenceUpdate(Pref pref);
   async OpenVRControllerActionPathToVR(nsCString aPath);
   async OpenVRControllerManifestPathToVR(OpenVRControllerType aType, nsCString aPath);
   async RequestMemoryReport(uint32_t generation,